src/hotspot/os/windows/os_windows.cpp

```diff
@@ -1280,7 +1280,15 @@ void  os::dll_unload(void *lib) {
 }
 
 void* os::dll_lookup(void *lib, const char *name) {
-  return (void*)::GetProcAddress((HMODULE)lib, name);
+  ::GetLastError(); // Clear old pending errors
```

Comment:
There is nothing in the specification for `GetLastError` to indicate that it will clear the error itself. If you want to do this then you need to call `SetLastError(0)`.

Reply:
You are correct, thanks!